roles/installer/defaults/main.yml

```diff
@@ -319,6 +315,12 @@ redis_resource_requirements:
   requests:
     cpu: 50m
     memory: 64Mi
+ 
```

Comment:
It looks like these are the only errors left:

```
|./roles/installer/defaults/main.yml
  Error: 318:1 [trailing-spaces] trailing spaces
  Error: 323:1 [trailing-spaces] trailing spaces
```

Reply:
@rooftopcellist  just fixed those errors